File: deleteatbeginarray.cpp
# include <iostream>
using namespace std;
int main()
{
    int array[20], n ;
    cout<< "Enter size of the array: ";
    cin>> n;
    cout<< "enter array elements: ";
    for (int i=0 ; i<n ; i++)
    {
        cin>> array[i];
    }
    for (int i=0 ; i<n-1 ; i++){
        array[i] = array[i+1];
    }
    n--;
    cout<< "array after deletion: " <<endl;
    for (int i=0 ; i<n; i++ )
    {
        cout<< array[i] <<endl;
    }
    return 0;
}

File: deleteatbeginarray.java
import java.util.Scanner;

public class deleteatbeginarray {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int[] array = new int[20];
        int n;

        System.out.print("Enter size of the array: ");
        n = scanner.nextInt();

        System.out.print("Enter array elements: ");
        for (int i = 0; i < n; i++) {
            array[i] = scanner.nextInt();
        }

        for (int i = 0; i < n - 1; i++) {
            array[i] = array[i + 1];
        }
        n--;

        System.out.println("Array after deletion:");
        for (int i = 0; i < n; i++) {
            System.out.println(array[i]);
        }
    }
}
